src/System.Management.Automation/engine/Modules/ModuleCmdletBase.cs

```diff
@@ -2475,6 +2536,10 @@ internal bool LoadModuleManifestData(ExternalScriptInfo scriptInfo, ManifestProc
             manifestInfo.ProcessorArchitecture = requiredProcessorArchitecture;
             manifestInfo.Prefix = resolvedCommandPrefix;
 
+            // A module is considered compatible if it's not on the System32 module path, or
+            // if it is and declared "Core" as a compatible PSEdition.
+            manifestInfo.IsConsideredEditionCompatible = isConsideredCompatible;
```

Comment:
`IsConsideredEditionCompatible` should reflect the truth about whether a module is considered compatible, which shouldn't be affected by `BaseSkipEditionCheck`.

Reply:
Ah good catch!